### PR TITLE
Feature/FOUR-12634: The ellipsis called "Edit Launchpad' needs to open the lauchpad settings

### PR DESCRIPTION
**Pull request by CarliPinell**
## Issue & Reproduction Steps
This ticket has next improvements:
* Addition of modified ModalSaveVersion Component into SharedComponents folder
* New ellipsis link in Launchpad page called "Edit in Launchpad"
* Backward compatibility ModalSaveVersion component with package-versions PUBLISH button

## How to Test
- Go to Designer
- Click on Processes icon
- Select a process , go to ellipsis menu and choose Open in Launchpad
- Click on Ellipsis Menu and select "Edit in Launchpad"

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-12634

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:deploy
ci:next

### CODE REVIEW

File: resources/js/processes-catalogue/components/ProcessMap.vue [hunk 1]
```diff
@@ -76,10 +86,16 @@ export default {
...
+      optionsData: {},
     };
   },
   mounted() {
     this.getActions();
+    this.optionsData = {
+      id: this.process.id.toString(),
+      type: "Process",
+    };
+    console.log('this.optionsData ProcesMap: ', this.optionsData);
```
Comment (pmPaulis) [inline resources/js/processes-catalogue/components/ProcessMap.vue:98]: Remove console log

**Review by pmPaulis — CHANGES_REQUESTED** [review]

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/processes-catalogue/components/ProcessMap.vue (hunk 1 above)
Comment (CarliPinell) [reply resources/js/processes-catalogue/components/ProcessMap.vue:98]: solved

File: resources/js/components/shared/IconDropdown.vue [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+              <i class="fas fa-fw" :class="'fa-'+props.option.value"></i>
+            </div>
+          </template>
+        </multiselect>
+      </b-input-group>
+    </div>
+</template>
+
+<script>
+    import Icons from './Icons';
```
[the remark this answers is not in the document]
Comment (CarliPinell) [reply resources/js/components/shared/IconDropdown.vue:42]: solved

File: resources/js/components/shared/IconDropdown.vue [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+        },
+        beforeMount() {
+          this.list = this.all;
+        },
+        mounted() {
+            if (this.value) {
+              this.icon = this.find(this.value);
+            } else {
+              this.icon = this.find(this.default);
+            }
```
[the remark this answers is not in the document]
Comment (CarliPinell) [reply resources/js/components/shared/IconDropdown.vue:81]: solved

File: resources/js/components/shared/IconDropdown.vue [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+        },
+        methods: {
+          onSearch(query) {
+            if (query.length) {
+              this.query = query.toLowerCase();
+            } else {
+              if (this.isOpen) {
+                this.query = '';
+              }
+            }
```
[the remark this answers is not in the document]
Comment (CarliPinell) [reply resources/js/components/shared/IconDropdown.vue:91]: solved

File: resources/js/bootstrap.js [hunk 5]
```diff
@@ -27,6 +27,8 @@ import AccessibilityMixin from "./components/common/mixins/accessibility";
 import PmqlInput from "./components/shared/PmqlInput.vue";
 import DataTreeToggle from "./components/common/data-tree-toggle.vue";
 import TreeView from "./components/TreeView.vue";
+import ModalSaveVersion from "./components/shared/ModalSaveVersion.vue";
+import IconDropdown from "./components/shared/IconDropdown.vue";
```
[the remark this answers is not in the document]
Comment (CarliPinell) [reply resources/js/bootstrap.js:31]: solved

**Pull request merged**